src/settlements/settlements.controller.ts

```diff
@@ -133,6 +133,31 @@ export class SettlementsController {
     return handleServiceResult(result);
   }
 
+  @ApiTags('instructions')
+  @ApiOperation({
+    summary: 'Withdraw affirmation from an existing Instruction',
+    description: 'This endpoint will withdraw an affirmation from an Instruction',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'The ID of the Instruction from which to withdraw the affirmation',
+    type: 'string',
+    example: '123',
+  })
+  @ApiOkResponse({
+    description: 'Details of the transaction',
+    type: TransactionQueueModel,
+  })
```

Comment:
Add ApiNotFoundResponse as well for the case where no instruction is found for given ID